src/lib/intercom/index.js

```diff
@@ -0,0 +1,20 @@
+import { resize } from './resize'
+import { branding } from './branding'
+
+const process = event => {
+  if (resize.canHandle(event)) {
+    resize.handle(event)
+    return true
+  }
+
+  if (branding.canHandle(event)) {
+    branding.handle(event)
+    return true
+  }
+
+  return false
```

Comment:
if this pattern keeps expanding please use the full chain of resp pattern https://refactoring.guru/design-patterns/chain-of-responsibility